apps/hash-agents/pyproject.toml

```diff
@@ -15,6 +15,13 @@ openai = "^0.27.4"
 python-dotenv = "^1.0.0"
 tiktoken = "^0.3.3"
 
+[tool.poetry.group.dev.dependencies]
+black = "^23.3.0"
+
+[tool.black]
+line-length = 100
```

Comment:
A specific reason why a non-standard line length was chosen? The default line length of 88 is already higher than the 79 that PEP8 enforces.

Reply:
might make sense to also opt into [preview mode](https://black.readthedocs.io/en/stable/the_black_code_style/future_style.html#preview-style), especially parentheses in with expressions is very nice.

Reply:
We use `100` in other places, but it's probably good to stick with the default for now. It's easy to change it later.

I like the preview-mode, thanks for the suggestion!
Feel free to suggest more 🚀